versions.json

```diff
@@ -35,8 +35,9 @@
         "fpm_directories": ["bin","include","lib"]
     },
     "aws-sdk-cpp": {
-        "commitish": "1.4.89",
-        "version_string": "1.4.89",
+        "github_repo": "https://github.com/aws/aws-sdk-cpp",
+        "commitish": "1.9.185",
+        "version_string": "1.9.185",
```

Comment:
requires pulling in latest changes from upstream.

Reply:
i've updated irods/aws-sdk-cpp, it's now in sync with upstream main.

Reply:
removed `"github_repo"`